alert/alertlog/store.go

```diff
@@ -339,6 +341,7 @@ func (s *Store) logAny(ctx context.Context, tx *sql.Tx, insertStmt *sql.Stmt, id
 			case notification.DestTypeUserEmail:
 				r.subject.classifier = "Email"
 			case notification.DestTypeUserWebhook:
+			case notification.DestTypeWebhook:
```

Comment:
Go switch statements have an implicit `break`, so we need to use `fallthrough` here explicitly.
```suggestion
			case notification.DestTypeWebhook:
				fallthrough
			case notification.DestTypeUserWebhook:
```